jaxrs-tck/src/main/java/jakarta/ws/rs/tck/contextprovider/jsonb/CustomJsonbProvider.java

```diff
@@ -0,0 +1,53 @@
+package jakarta.ws.rs.tck.contextprovider.jsonb;
```

Comment:
Why did you separate my *deliberately* inner classes into *public* classes?

Reply:
Sorry, I usually don't put that much into one file, so it felt a bit more comfy like that. But I agree, I could have at least made them package-protected. But I'll try to change it back the way it was if you like.

Reply:
I put the inner classes the way they were.